tools/src/schedule/debian.rs

```diff
@@ -32,8 +33,13 @@ impl SourcePkgBucket {
         }
     }
 
-    pub fn get(&self, pkg: &DebianBinPkg) -> Result<&DebianSourcePkg> {
+    pub fn get(&self, pkg: &DebianBinPkg) -> Result<DebianSourcePkg> {
         let (name, version) = &pkg.source;
+        let re = Regex::new(r"^.*+b\d+$").unwrap();
+        let mut bin_nmu = "".to_string();
+        if let Some(caps) = re.captures(&pkg.version) {
+            bin_nmu = caps[1].to_string();
+        }
```

Comment:
Maybe this could be written as:
```rust
let bin_nmu = pkg.version.rsplit_once("+b")
    .map(|(_prefix, suffix)| suffix)
    .unwrap_or("");
```

If you want to ensure it's a number you could add this between `map` und `unwrap_or`:
```rust
.filter(|num| num.parse::<u64>().is_ok())
```

Reply:
nice, thx.